test/SparkDeployBase.t.sol

```diff
@@ -77,75 +77,86 @@ abstract contract SparkDeployBaseTest is Test {
+
         assertEq(poolAddressesProviderRegistry.getAddressesProviderIdByAddress(address(poolAddressesProvider)), 1);
     }
 
     function test_spark_deploy_poolAddressesProvider() public {
-        assertEq(poolAddressesProvider.owner(), admin);
-        assertEq(poolAddressesProvider.getMarketId(), "Spark Protocol");
-        assertEq(poolAddressesProvider.getPool(), address(pool));
-        assertEq(poolAddressesProvider.getPoolConfigurator(), address(poolConfigurator));
-        assertEq(poolAddressesProvider.getPriceOracle(), address(aaveOracle));
-        assertEq(poolAddressesProvider.getACLManager(), address(aclManager));
-        assertEq(poolAddressesProvider.getACLAdmin(), admin);
+        assertEq(poolAddressesProvider.owner(),                  admin);
+        assertEq(poolAddressesProvider.getMarketId(),            "Spark Protocol");
+        assertEq(poolAddressesProvider.getPool(),                address(pool));
+        assertEq(poolAddressesProvider.getPoolConfigurator(),    address(poolConfigurator));
+        assertEq(poolAddressesProvider.getPriceOracle(),         address(aaveOracle));
+        assertEq(poolAddressesProvider.getACLManager(),          address(aclManager));
+        assertEq(poolAddressesProvider.getACLAdmin(),            admin);
         assertEq(poolAddressesProvider.getPriceOracleSentinel(), address(0));
-        assertEq(poolAddressesProvider.getPoolDataProvider(), address(protocolDataProvider));
+        assertEq(poolAddressesProvider.getPoolDataProvider(),    address(protocolDataProvider));
     }
 
     function test_spark_deploy_aclManager() public {
         // NOTE: Also verify that no other address than the admin address has any role (verify with events)
         assertEq(address(aclManager.ADDRESSES_PROVIDER()), address(poolAddressesProvider));
-        assertTrue(aclManager.hasRole(aclManager.DEFAULT_ADMIN_ROLE(), admin));
-        assertTrue(!aclManager.hasRole(aclManager.DEFAULT_ADMIN_ROLE(), deployer));
-        assertEq(aclManager.getRoleAdmin(aclManager.POOL_ADMIN_ROLE()), aclManager.DEFAULT_ADMIN_ROLE());
-        assertTrue(aclManager.hasRole(aclManager.POOL_ADMIN_ROLE(), admin));
+
+        bytes32 defaultAdmin = aclManager.DEFAULT_ADMIN_ROLE();
+
+        assertEq(aclManager.getRoleAdmin(aclManager.POOL_ADMIN_ROLE()),      defaultAdmin);
+        assertEq(aclManager.getRoleAdmin(aclManager.EMERGENCY_ADMIN_ROLE()), defaultAdmin);
+
+        assertTrue( aclManager.hasRole(defaultAdmin, admin));
+        assertTrue(!aclManager.hasRole(defaultAdmin, deployer));
+
+        assertTrue( aclManager.hasRole(aclManager.POOL_ADMIN_ROLE(), admin));
         assertTrue(!aclManager.hasRole(aclManager.POOL_ADMIN_ROLE(), deployer));
-        assertEq(aclManager.getRoleAdmin(aclManager.EMERGENCY_ADMIN_ROLE()), aclManager.DEFAULT_ADMIN_ROLE());
+
         if (block.chainid == 1) assertTrue(aclManager.hasRole(aclManager.EMERGENCY_ADMIN_ROLE(), admin));     // FIXME missing on GOERLI
```

Comment:
This conditional should be removed. There shouldn't be discrepencies between chains.

Reply:
test